qiskit/transpiler/passes/optimization/split_2q_unitaries.py

```diff
@@ -36,5 +37,21 @@ def __init__(self, fidelity: float = 1.0 - 1e-16):
 
     def run(self, dag: DAGCircuit) -> DAGCircuit:
         """Run the Split2QUnitaries pass on `dag`."""
-        split_2q_unitaries(dag, self.requested_fidelity)
-        return dag
+        result = split_2q_unitaries(dag, self.requested_fidelity)
```

Comment:
Perhaps the docstring of the `Split2QUnitaries` class (lines 22-28) should also be updated?

Reply:
Done in 1d7b4f61a53c96a56090d392a6ca0423dc48ea20